src/packages/shared-utils/package-check.ts

```diff
@@ -11,13 +12,8 @@ const secondClockStatuses = [
   SEATOOL_STATUS.PENDING_CONCURRENCE,
 ];
 
-const checkAuthority = (
-  authority: Authority | null,
-  validAuthorities: Authority[],
-) =>
-  !authority
-    ? false
-    : validAuthorities.includes(authority.toLowerCase() as Authority);
+const checkAuthority = (authority: string, validAuthorities: Authority[]) =>
+  validAuthorities.some((validAuthority) => validAuthority === authority);
```

Comment:
Since TS won't allow us to use `.include` with typed out arrays, the workaround is either casting (yuck) or using `.some` to determine whether a untyped value (such as a `string`) exists within an array